src/VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -76,6 +78,23 @@ public Startup(IConfiguration configuration, IWebHostEnvironment hostingEnvironm
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            // https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer?view=aspnetcore-3.1#forward-the-scheme-for-linux-and-non-iis-reverse-proxies
+            if (string.Equals(
```

Comment:
I'm not sure what this change will work without the addition `app.UseForwardedHeaders();` in the `Configure()` method of statup.cs
See the same code in the storefront.
https://github.com/VirtoCommerce/vc-demo-storefront/pull/40/files/c08f5e64fe7b557d81471aee215280944ccb16f9#diff-94e39c15f105756c1423a7ad1702f381d6ae7bae543a6aa1b635e2c030000940